File: NetBeans_IDE/JsonPL/src/com/middlemind/JsonPL/SystemFunctionHandler.java
package com.middlemind.JsonPL;

import com.middlemind.JsonPL.JsonObjs.JsonObjSysBase;
import java.util.List;

/**
 *
 * @author Victor G. Brusca, Middlemind Games 03/28/2022 06:25 PM EDT
 */
public interface SystemFunctionHandler {

   public JsonObjSysBase call(String name, List<JsonObjSysBase> args);
}

File: storage/alt_interpreters/VisualStudio_IDE/JsonPL/JsonPL/com/middlemind/JsonPL/SystemFunctionHandler.cs
using com.middlemind.JsonPL.JsonObjs;
using System;
using System.Collections.Generic;
using System.Text;

namespace com.middlemind.JsonPL {
   /**
   *
   * @author Victor G. Brusca, Middlemind Games 03/28/2022 06:25 PM EDT
   */
   public interface SystemFunctionHandler {
      public JsonObjSysBase call(string name, List<JsonObjSysBase> args);
   }
}
